src/test/java/com/google/alpollo/AnalysisTest.java

```diff
@@ -44,10 +57,42 @@ public void checkSentimentScore() throws IOException {
 
   @Test
   public void checkSentimentMagnitude() throws IOException {
-    Sentiment sentiment = helper.getSentiment(LYRICS);
+    Sentiment sentiment = helper.getSentiment(LYRICS_LONG);
     float actual = sentiment.getMagnitude();
     float expected = 0.7f;
     
     Assert.assertEquals(expected, actual, 0.05f);
   }
+
+  /**
+   * If the lyrics are too short, the API might find less than 10 entities.
```

Comment:
Initially I added everything because I wanted to check if everything works correctly. I could make a separate test to see if `getEntityList()` and use a manually defined list here.